string goal_label
---
geometry_msgs/Quaternion goal_heading
bool success